test/app/Gemfile

```diff
@@ -1,13 +1,16 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-gem 'rails', '>= 6', '< 7'
+gem 'rails', '~> 7.0.0'
 gem 'sqlite3', '~> 1.4'
 gem 'puma'
 gem 'bootsnap', require: false
 gem 'psych', '< 4' # for safe_load degration
```

Comment:
I guess this is needed for `< Rails 6.1.4`.

> Fix compatibility with psych >= 4.
> https://github.com/rails/rails/releases/tag/v6.1.4

Therefore, this can be removed.

Reply:
Removed.